src/libdrakvuf/drakvuf.c

```diff
@@ -659,6 +659,14 @@ addr_t drakvuf_get_kernel_base(drakvuf_t drakvuf)
     return drakvuf->kernbase;
 }
 
+addr_t drakvuf_get_kernel_kaslr(drakvuf_t drakvuf)
+{
+    addr_t _text;
+    if (!drakvuf_get_kernel_symbol_rva(drakvuf, "_text", &_text))
```

Comment:
This seems to be Linux specific yet it's not abstracted into an OS-level function.